src/pages/ServiceDetails/ServiceInfo/ServiceInfoPods.tsx

```diff
@@ -1,66 +1,97 @@
 import * as React from 'react';
-import { Col, Row } from 'patternfly-react';
-import LocalTime from '../../../components/Time/LocalTime';
+import { Col, Row, OverlayTrigger, Tooltip } from 'patternfly-react';
 import Badge from '../../../components/Badge/Badge';
 import { Pod } from '../../../types/ServiceInfo';
 import { PfColors } from '../../../components/Pf/PfColors';
+import { PodsGroup, groupPods } from './ServiceInfoPodsGrouping';
 
 interface Props {
   pods?: Pod[];
 }
 
 class ServiceInfoPods extends React.Component<Props> {
+  groups: PodsGroup[];
+
   constructor(props: Props) {
     super(props);
+    if (props.pods) {
+      this.groups = groupPods(props.pods);
+    } else {
+      this.groups = [];
+    }
   }
 
   render() {
     return (
       <div className="card-pf">
         <Row className="row-cards-pf">
           <Col xs={12} sm={12} md={12} lg={12}>
-            {(this.props.pods || []).map((pod, u) => (
+            {(this.groups || []).map((group, u) => (
               <div className="card-pf-body" key={'pods_' + u}>
-                <h3>{pod.name}</h3>
+                <h3>
+                  {group.numberOfPods > 1 ? (
+                    <OverlayTrigger
+                      // prettier-ignore
```

Comment:
Instead to add a prettier-ignore, better to modify the prettier rules ?

Reply:
Prettier is sometimes tedious to configure and afaik there's no granularity in how rules are applied, it's just "on" or "off". Here prettier decides that the line should be split after `{`, but tslint refuses spaces after `{`, so they clash.

Personally I would like to completely disable prettier fanciful line-breakings, it's quite bad at that game. But when I suggested that a while ago I didn't get much positive votes :)

Reply:
I don't have much knowledge about prettier, but I guess that we should do something when we find that is a pain for many of us. I think we should react when we have some more occurrences.

Reply:
What we could do is:
- continue to add "prettier-ignore" commands as we found issues
- when it becomes obvious that prettier is not performing well (for instance, after 3 issues with the same rule) then opt-out the rule.